wp-content/themes/missilethreat/template-parts/block-post.php

```diff
@@ -14,6 +14,12 @@
 ?>
 
 <article <?php post_class('post-block post-block--post'); ?> id="post-<?php the_ID(); ?>">
+<?php
+	if ( has_post_thumbnail()) : ?>
+    <a href="<?php the_permalink(); ?>" alt="<?php the_title_attribute(); ?>" class="thumbnail-link">
```

Comment:
You can remove the `alt` tag from the `<a>` tag. It'll be included on the image itself. However, it might be worth putting `aria-hidden="true"` (look up the correct syntax) on this, since its superfluous to the title being a link to the same page. Since the image is there for decoration, we can tell screen readers to ignore it.